controller/migrations/data/0001.sh

```diff
@@ -1,19 +1,15 @@
 #!/usr/bin/env bash
 
-ETCD_PORT=${ETCD_PORT:-4001}
-ETCD="$HOST:$ETCD_PORT"
```

Comment:
Looking through the rest of this PR, this will definitely need to stay the same for older clusters without ETCDCTL_PEERS set up.